scripts/create_patches_files.py

```diff
@@ -51,16 +63,33 @@ def parse_patches(patches, outdir="./patch_urls"):
     with open("./artifact_names.txt", "w") as f:
         f.write(str(artifact_names))
 
-def get_patches(start: str, end: str):
-    url = "https://patchwork.sourceware.org/api/1.3/patches/?order=-date&q=RISC-V&project=6&before={}&since={}"
+def get_patches(start: str, end: str, backup: str):
+    url = "https://patchwork.sourceware.org/api/1.3/patches/?order=date&q=RISC-V&project=6&before={}&since={}"
```

Comment:
I'd reorder the before/since to make the `.format()` call a bit more intuitive to read